### ISSUE TITLE
Increase test timeout for intermittent e2e CI failures

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2642]
We sometimes fail our e2e tests on CI:

```
FAIL browser: chromium src/api/auth/test/e2e/signup-flow.test.js (17.059 s)
  Signup Flow
    ✕ should have none of the users in the Users service for test data accounts (10042 ms)
    ✓ signup flow works to login and register a new Telescope user (2138 ms)
    ✓ signup flow fails if user is not authenticated (236 ms)
    ✓ signup flow fails if user data is missing required properties (959 ms)
    ✓ signup flow fails if user is already a Telescope user (1108 ms)

  ● Signup Flow › should have none of the users in the Users service for test data accounts

    thrown: "Exceeded timeout of 10000 ms for a hook.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      38 |   });
      39 |
    > 40 |   beforeEach(async () => {
         |   ^
      41 |     await cleanupTelescopeUsers(users);
      42 |     context = await browser.newContext();
      43 |     page = await browser.newPage();

      at src/api/auth/test/e2e/signup-flow.test.js:40:3
      at Object.<anonymous> (src/api/auth/test/e2e/signup-flow.test.js:34:1)
```

I suspect that this is resource constraints on the VMs we get from GitHub for Actions, and if we gave it more time before it timed-out we'd be OK.

Let's try bumping https://github.com/Seneca-CDOT/telescope/blob/master/jest.config.base.js#L12 to 20s or something.

**Comment by humphd** [comment]
I just hit this today, see:

- https://github.com/Seneca-CDOT/telescope/actions/runs/1698210094
- https://github.com/Seneca-CDOT/telescope/runs/4818283866?check_suite_focus=true

**Comment by humphd** [comment]
@DerekJxy this didn't get closed.  You used `Fixes issue #2642` vs. `Fixes #2642`.  The specific format matters.  Next time, use `Fixes #...`
